<launch>
	<node pkg="alpheus_actions" type="headingServer.py" name="headingServer" output="screen"/>
	<node pkg="tests" type="headingTest.py" name="headingTest" output="screen"/>
	<node pkg="mission_planner" type="mission_planner.py" name="mission_planner" output="screen"/>
</launch>
